transformer_lens/components/attention.py

```diff
@@ -0,0 +1,569 @@
+"""Hooked Transformer Attention Component.
```

Comment:
This is missing the latest alibi code - I think it was probably a change since your PR